server/src/messages/messages.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@nestjs/common';
+import { Message } from './entities/message.entity';
+import { User } from '../users/entities/user.entity';
+import { Repository } from 'typeorm';
```

Comment:
should be `import type {...`

Reply:
I get this when trying to start up the app (`npm run start:dev`) after this change

![image](https://user-images.githubusercontent.com/40244233/142339483-38f9f637-b7d0-48eb-8b2d-21e44ee67942.png)

```
[2:16:13 AM] Starting compilation in watch mode...

src/messages/messages.service.ts:9:2 - error TS1361: 'Injectable' cannot be used as a value because it was imported using 'import type'.

9 @Injectable()
   ~~~~~~~~~~

  src/messages/messages.service.ts:1:15
    1 import type { Injectable } from '@nestjs/common';
                    ~~~~~~~~~~
    'Injectable' was imported here.

src/messages/messages.service.ts:12:23 - error TS1361: 'Message' cannot be used as a value because it was imported using 'import type'.

12     @InjectRepository(Message) private messagesRepository: Repository<Message>,
                         ~~~~~~~

  src/messages/messages.service.ts:2:15
    2 import type { Message } from './entities/message.entity';
                    ~~~~~~~
    'Message' was imported here.

[2:17:31 AM] Found 2 errors. Watching for file changes.
```

Reply:
sorry, just meant for `Repository`

Reply:
and `DeleteResult` 
the things we're using as types basically
not a biggie